core/src/main/scala/org/apache/spark/storage/DiskStore.scala

```diff
@@ -84,11 +84,11 @@ private[spark] class DiskStore(
         }
       }
     }
-    val finishTime = System.currentTimeMillis
-    logDebug("Block %s stored as %s file on disk in %d ms".format(
+    val finishTimeNs = System.nanoTime()
+    logDebug("Block %s stored as %s file on disk in %d ns".format(
```

Comment:
Same comment about logging in ms here.
You are welcome to convert this to string interpolation too.